### PR TITLE
fix(Tio Anime)

### PR DESCRIPTION
**Pull request by u0f**
**Describe the changes in this pull request:**

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

</details>

### CODE REVIEW

File: websites/T/Tio Anime/presence.ts [hunk 1]
```diff
@@ -21,8 +21,8 @@ presence.on("UpdateData", async () => {
       largeImageKey: "tioanimelogo",
       startTimestamp: browsingTimestamp
     },
-    privacy = await presence.getSetting<boolean>("privacy"),
```
Comment (down-bad) [inline websites/T/Tio Anime/presence.ts:24]: Why'd you remove the type?

File: websites/T/Tio Anime/presence.ts [hunk 2]
```diff
@@ -21,8 +21,8 @@ presence.on("UpdateData", async () => {
...
-    buttons = await presence.getSetting<boolean>("buttons");
```
Comment (down-bad) [inline websites/T/Tio Anime/presence.ts:25]: Why'd you remove the type?

Comment (u0f) [reply websites/T/Tio Anime/presence.ts:25]: oh, mb

**Review by Bas950 — DISMISSED** [review]
bump version

**Comment by ImRodry** [comment]
Title doesn't follow commitlint @EncryptedDev

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**